tests/main/lxd-postrm-purge/task.yaml

```diff
@@ -0,0 +1,82 @@
+summary: Check that package remove and purge works inside LXD containers
+
+# Since it's only apt remove --purge and lxd tests are rather long, limit to a
+# couple of systems only. The postrm purge is more thoroughly checked in
+# tests/main/postrm-purge.
+systems: [ubuntu-18.04-*, ubuntu-20.04-*]
+
+# lxd downloads can be quite slow
+kill-timeout: 25m
+
+# start early
+priority: 1000
+
+prepare: |
+    # using apt here is ok because this test only runs on ubuntu
+    echo "Remove any installed debs (some images carry them) to ensure we test the snap"
+    # apt -v to test if apt is usable (its not on ubuntu-core)
```

Comment:
this test only runs on ubuntu classic though, should this test systems be expanded to also run on ubuntu-core too ?

Reply:
We need to have a deb to push to the container.

Reply:
In that case perhaps reword the test to not mention ubuntu-core since this test doesn't run on ubuntu-core ?